edit-post/components/sidebar/featured-image/index.js

```diff
@@ -30,7 +23,7 @@ function FeaturedImage( { isOpened, postType, onTogglePanel } ) {
 			<PanelBody
 				title={ get(
 					postType,
-					[ 'data', 'labels', 'featured_image' ],
+					[ 'labels', 'featured_image' ],
```

Comment:
Can we also `camelCaseDeepKeys` all keys in the future? :)

I have it implemented in coediting PR: https://github.com/WordPress/gutenberg/blob/35df0a590c0230519ceb36616d2859ed1d0c8888/coediting/utils/index.js

We could apply `camelCaseDeepKeys` to all responses as part of the resolver processing.

Reply:
If we go this direction, I don't think we should apply `camelCaseDeepKeys` arbitrarily but more create well defined models for all our entities (we don't have to match the API object at all). I think this is a large subject we should probably discuss separately and come up with a plan.

Reply:
Oh yes, definitely not for this PR. I need to be more cautious when leaving random thoughts and state it clearly that I just want to open an unrelated discussion :)

We can defer this one, but it would be nice to have well-defined models validated with JSON schema. Ideally automated 😃